scripts/install-libsodium.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+
+mkdir -p lib
+
+curl https://download.libsodium.org/libsodium/releases/libsodium-1.0.12.tar.gz \
+    | tar xz --directory lib/
+
+pushd lib/libsodium-1.0.12/
+
+./configure
+make
+make check
+sudo make install
+
+popd
```

Comment:
I updated this to add the `mkdir -p lib` command as otherwise it failed on my machine. While I was at it, I spaced it a bit more and added the `set -e` at the top.